src/app/screens/ReceiveInvoice/index.tsx

```diff
@@ -142,8 +142,15 @@ function ReceiveInvoice() {
     if (!invoice) return null;
     return (
       <>
-        <div className="mt-4 relative p-8 bg-white rounded-lg shadow-sm ring-1 ring-black ring-opacity-5 flex justify-center items-center overflow-hidden">
-          <QRCode value={invoice.paymentRequest.toUpperCase()} />
+        <div className="mt-4 relative p-8 bg-white dark:bg-surface-01dp rounded-lg shadow-sm ring-1 ring-black ring-opacity-5 flex justify-center items-center overflow-hidden">
+          <div className="relative flex items-center justify-center">
+            <QRCode value={invoice.paymentRequest.toUpperCase()} />
```

Comment:
I think we missed this, it also shows the Alby QR code for Alby accounts. There it should show the Alby user's avatar if they have one.

See src/app/screens/Receive/index.tsx for the code that is used to display the Alby user avatar.

I see if the Alby user does not have an `avatarUrl` it'll use the randomly generated avatar. I think showing the Alby icon would be better here. @reneaaron @stackingsaunter what do you think?

Reply:
Okay, then I will make changes for the first one.

Reply:
It looks good to me, apart from a bit of duplicated code. But maybe that can be cleaned up later. Thanks @Rithvik-padma !